apps/server/src/controllers/riderControllers/index.ts

```diff
@@ -0,0 +1 @@
+export * from './authRiders'
```

Comment:
How about exporting `getRiderDetails.ts` as a module so it can export all available functions from the file?

Reply:
Okay